aws_infra/glue.tf

```diff
@@ -0,0 +1,14 @@
+# Create glue catalogue database
+resource "aws_glue_catalog_database" "dev_dataplatform_database" {
+  name        = "dev-male-female-ingestion"
+  description = "dev database for data platform team for random profiles"
+
+  tags = {
+    Name        = "Data platform database"
+    Environment = "Dev"
+    owner       = "blessing_and_leo"
+    team        = "Data Platform Team"
+    managed_by  = "Mayowa"
```

Comment:
Managed by Terraform is better, so when someone see it, they know its a resource provisioned with terraform